e2e-nodejs/group-connection/test-connection-multi-network.mjs

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+import { success, fail, testThis, log } from '../../tools/scripts/utils.mjs';
+import { LitNodeClient } from '@lit-protocol/lit-node-client';
+
+export async function main() {
+  const networks = ['cayenne', 'habanero', 'manzano'];
+
+  for (const network of networks) {
+    // ==================== Test Logic ====================
+    const client = new LitNodeClient({
+      litNetwork: network,
+      debug: globalThis.LitCI.debug,
+    });
+    log(`connecting to ${network.toUpperCase()}...`);
+    await client.connect();
+
+    // ==================== Post-Validation ====================
+    if (!client.ready) {
+      return fail('client not ready');
+    }
+    if (client.config.litNetwork !== network) {
```

Comment:
Could we add a check on `globalThis` to assert on the new global namespacing?

Reply:
I didn't implement the global namespacing, I think it's a risky approach (see also https://github.com/LIT-Protocol/js-sdk/pull/331#issuecomment-1916891635). There could be a use case for global namespaces I'm not aware of yet: in that case please let me know.